scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/vqfn.yaml

```diff
@@ -561,6 +561,50 @@ VQFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+VQFN-32-1EP_5x5mm_P0.5mm_EP3.15x3.15mm:
+  device_type: 'VQFN'
+  size_source: 'https://www.ti.com/lit/ds/slvs589d/slvs589d.pdf#page=33'
+  ipc_class: 'qfn'
+  body_size_x:
+    minimum: 4.85
+    nominal: 5
+    maximum: 5.15
+  body_size_y:
+    minimum: 4.85
+    nominal: 5
+    maximum: 5.15
+  overall_height:
+    minimum: 0.8
+    maximum: 1
+  lead_width:
+    minimum: 0.18
+    maximum: 0.3
+  lead_len:
+    minimum: 0.3
+    maximum: 0.5
+
+  EP_size_x:
+    minimum: 2.95
+    nominal: 3.15
+    maximum: 3.25
```

Comment:
You can also write that as:
```yaml
EP_size_x: 3.15 +/- 0.1
```
But this format is also okay.

Reply:
Thanks, your format is better, I have updated it

Reply:
I noticed a mistake in EP_size_y. I have corrected this and also changed a few other lines to match this more compact formatting.